apps/src/craft/code-connection/craft.js

```diff
@@ -138,6 +242,8 @@ export default class Craft {
       resetButton.style.minWidth = runButton.offsetWidth + 'px';
     }
 
+    // Turn on call tracing
+    Blockly.mainBlockSpace.traceOn(true);
```

Comment:
Seems like `studioApp().highlight(blockID);` doesn't work without this boolean change